Bottom: dad10e6ff855cc6568ff327339beff31db2ab24b
Top:    af06d10c6d5b2a4a1ac2f36d3bc27ccc6b23d440
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index ea95b10..08b4d56 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1284,6 +1284,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
    *
    */
 
+    fprintf (stderr, "Subtracting pin + via drills\n");
+    r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+    r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
   if (info.poly == NULL)
     {
       fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
